assets/src/components/site-scan-results/style.scss

```diff
@@ -51,12 +51,15 @@
 
 .site-scan-results__source {
 	align-items: center;
-	display: flex;
-	flex-flow: row nowrap;
 	font-size: 14px;
 	margin: 0;
 	min-height: 3.5rem;
 	padding: 1rem;
+	max-width: 100%;
```

Comment:
I've noticed some CSS bugs caused by the latest changes. Text in a row is not vertically aligned to the middle. Also, the arrow doesn't rotate:

https://user-images.githubusercontent.com/478735/155304381-3b1e1406-4f01-4c9d-94e5-31762c5f4e91.mp4

I've played around with it and was able to fix those issues. Here's a patch you can use if you wish:

```diff
Index: assets/src/components/site-scan-results/style.scss
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/assets/src/components/site-scan-results/style.scss b/assets/src/components/site-scan-results/style.scss
--- a/assets/src/components/site-scan-results/style.scss	(revision fefdc7cafc8589fcf99532c1589d0ab91ec6cc62)
+++ b/assets/src/components/site-scan-results/style.scss	(date 1645612917416)
@@ -51,14 +51,15 @@
 
 .site-scan-results__source {
 	align-items: center;
+	display: flex;
 	font-size: 14px;
 	margin: 0;
-	min-height: 3.5rem;
-	padding: 1rem;
 	max-width: 100%;
+	padding: 1rem;
 
 	details {
 		margin: 0;
+		width: 100%;
 	}
 
 	&:nth-child(even) {
@@ -89,16 +90,18 @@
 
 .site-scan-results__source details summary {
 	display: flex;
+	align-items: center;
 }
 
 .site-scan-results__source details .dashicons {
+	align-items: center;
+	display: flex;
 	font-size: 14px;
-	margin-top: 2px;
+	justify-content: center;
 }
 
 .site-scan-results__source details[open] .dashicons {
-	rotate: 90deg;
-	margin: 0 5px 0 -5px;
+	transform: rotate(90deg);
 }
 
 .site-scan-results__source-version,
@@ -121,6 +124,7 @@
 }
 
 .site-scan-results__source-detail {
+	background-color: #fff;
 	border: 1px solid #dedede;
 	border-radius: 5px;
 	padding: 15px;
```

Here's how it works with the patch applied:

https://user-images.githubusercontent.com/478735/155304666-c0d32f14-93df-4919-bd5c-9b3f8be0364f.mp4

(Note that I've also added a white background to the `pre` element so that it's uniform no matter if it's in an even or an odd row.)